bundle/manifests/kepler.system.sustainable.computing.io_keplers.yaml

```diff
@@ -14,7 +14,29 @@ spec:
     singular: kepler
   scope: Cluster
   versions:
-  - name: v1alpha1
+  - additionalPrinterColumns:
+    - jsonPath: .spec.exporter.port
+      name: Port
+      type: integer
+    - jsonPath: .status.desiredNumberScheduled
+      name: Desired
+      type: integer
+    - jsonPath: .status.currentNumberScheduled
+      name: Current
+      type: integer
+    - jsonPath: .status.numberReady
+      name: Ready
+      type: integer
+    - jsonPath: .status.updatedNumberScheduled
+      name: Up-to-date
+      type: integer
+    - jsonPath: .status.numberAvailable
+      name: Available
+      type: integer
+    - jsonPath: .spec.exporter.nodeSelector
```

Comment:
print Tolerations also?

Reply:
I didn't see tolerations being print as part of kc get daemonset ... I am open to adding it though.

Reply:
I had some trouble adding toleration, default. I will update that in a separate PR.

Reply:
Created - https://github.com/sustainable-computing-io/kepler-operator/issues/186

Reply:
@vimalk78 , I have added tolerations to be print but the default one isn't listed (yet) which the issue above should fix.